packages/gatsby/src/db/loki/nodes.js

```diff
@@ -290,6 +293,63 @@ function deleteNodes(nodes) {
   }
 }
 
+const saveResolvedNodes = async (typeName, resolver, nodeTypeNames) => {
```

Comment:
do we use `typeName` here?

Reply:
No, because loki has separate tables, we store it via implementation type names.

Reply:
can we then remove `typeName` from the signature -- in the redux side of `saveResolvedNodes` we'd also want to use `nodeTypeNames` not `typeName` right?

Reply:
That is actually true. Good point.